content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -309,6 +309,9 @@ Here's one scenario where you would set it. Suppose the Container listens on 127
 and the Pod's `hostNetwork` field is true. Then `host`, under `httpGet`, should be set
 to 127.0.0.1. If your pod relies on virtual hosts, which is probably the more common
 case, you should not use `host`, but rather set the `Host` header in `httpHeaders`.
+kubelet will make the probe connection at the node but not in the pod which would want
```

Comment:
I don't think this sentence makes sense. Pods don't do probe checks. kubelet does probes on containers, not pods and certainly not services (which have the DNS name). When doing an `exec` check, the command is run in the container. When doing a HTTP or TCP check, it is essentially pod-level because all containers in the pod share the same IP. But the intent is that if you had multiple containers in there, you would put the health check network endpoint on different ports, and configure probes per container to talk to those different ports. That is, the liveness probe would *still* be per container.

I think that lines 9 of this document states pretty clearly "This page shows how to configure liveness and readiness probes for Containers." The "for Containers" part is critical. If you really want to drive the point home, you could add something like: "Kubelet can probe a container by executing a command in the container, making an HTTP request, or making a TCP connection. This check is intended to evaluate the liveness of the *container*, so when using a network probe, be sure that you specify a connection to the specific container. If multiple containers in the Pod have health network endpoints, they should be on different ports."

Reply:
It still relate to " how to configure liveness and readiness probes for Containers" while kubelet do it in the node level not in pod level.
Add this since there is no docs comment where kubelet do check, which could be confused for some users.